packages/components/app/styles/components/table.scss

```diff
@@ -36,8 +41,27 @@
   padding: 20px 16px;
 }
 
+
 .hds-table__thead {
-  border-bottom: 1px solid var(--token-color-border-primary);
+  .hds-table__tr {
+    color: #0c0c0e;
+    background-color: #f1f2f3;
```

Comment:
Looking at the design in Figma, these are standard colors. Better use design tokens (`--token-color-blah-blah`).

Reply:
plus, the background-color is already set in the `TH` elements, so it's useless here (try to set it red, and you will not see it). my suggestion is to delete the background color from here, move the `color` declaration in the `TH` elements, so you can also remove the border radius applied to the `TR` (see comment below)

Reply:
Good catch, updated.